app/Http/Controllers/PaymentController.php

```diff
@@ -135,59 +152,100 @@ public function proceed(Request $request){
+    //     return Redirect::route('home');
+    // }
+
     public function getPaymentStatus()
     {
-        /** Get the payment ID before session clear **/
-        $payment_id = Session::get('paypal_payment_id');
-        /** clear the session payment ID **/
-        Session::forget('paypal_payment_id');
-        if (empty(Input::get('PayerID')) || empty(Input::get('token'))) {
-            \Session::put('error','Payment failed');
-            return redirect()->route('payment')->with('error', 'Payment failed!');
-        }
-        $payment = Payment::get($payment_id, $this->_api_context);
-        /** PaymentExecution object includes information necessary **/
-        /** to execute a PayPal account payment. **/
-        /** The payer_id is added to the request query parameters **/
-        /** when the user is redirected from paypal back to your site **/
-        $execution = new PaymentExecution();
-        $execution->setPayerId(Input::get('PayerID'));
-        /**Execute the payment **/
-        $result = $payment->execute($execution, $this->_api_context);
-        /** dd($result);exit; /** DEBUG RESULT, remove it later **/
-        if ($result->getState() == 'approved') { 
-            $this->received();
-            /** it's all right **/
-            /** Here Write your database logic like that insert record or value in database if you want **/
-            \Session::put('success','Payment success');
-            return  Redirect::route('home');
-        }
-        DB::table('temp')->where('email', auth()->user()->email)->delete();
-       // $this->notReceived(Input::get('PayerID'));
-        \Session::put('error','Payment failed');
-        return Redirect::route('home');
+      $order=Order::where('id',Session::get('orderID'))->firstOrFail();
+      /** Get the payment ID before session clear **/
+      $payment_id = Session::get('paypal_payment_id');
+      $order->update([
+        'paypal'=>$payment_id,
+      ]);
+      /** clear the session payment ID **/
+      Session::forget('paypal_payment_id');
+      if (empty(Input::get('PayerID')) || empty(Input::get('token'))) {
+        \Session::put('error', 'Payment failed');
+        return redirect(url('/events'))->with('errorMessage','Something is wrong, please try again or contact with the admin.');
+
+      }
+      $payment = Payment::get($payment_id, $this->_api_context);
+      $execution = new PaymentExecution();
+      $execution->setPayerId(Input::get('PayerID'));
+      /**Execute the payment **/
+      $result = $payment->execute($execution, $this->_api_context);
+      if ($result->getState() == 'approved') {
+          \Session::put('success', 'Payment success');
+
+        //updating our order table on successfull checkout
+          $order->update([
+            'status'=>'payment succeeded',
+          ]);
+
+        //sending notification to admin and user
+        $arr=['order'=>$order];
+        Notification::route('mail', 'appnaok2018@gmail.com')
```

Comment:
Let's create another variables; $test_email ='stepholland99@gmail.com', $admin_email='stepholland99@gmail.com' 
and use those here instead. How does the user get sent an email?

Reply:
i used the admin mail. and user mail has been taken from the request of confirm events.